Dump theses scripts into a dedicated directory so they don't muck up bin.
They're too generically named, e.g.: deploy.py and project.py. Copies of Qt
tools like lrelease and rcc are also installed which we probably don't want
in PATH by default.

--- CMakeLists.txt.orig	2023-07-24 22:56:55 UTC
+++ CMakeLists.txt
@@ -62,7 +62,7 @@ else()
     foreach(file ${files})
         if(EXISTS ${file})
             install(FILES "${file}"
-                    DESTINATION bin
+                    DESTINATION bin/pyside6
                     PERMISSIONS
                     OWNER_EXECUTE OWNER_WRITE OWNER_READ
                     GROUP_EXECUTE GROUP_READ
@@ -74,7 +74,7 @@ else()
 
     foreach(directory ${directories})
         install(DIRECTORY "${directory}"
-                DESTINATION bin
+                DESTINATION bin/pyside6
                 FILE_PERMISSIONS
                 OWNER_EXECUTE OWNER_WRITE OWNER_READ
                 GROUP_EXECUTE GROUP_READ
